diagnostic.md

```diff
@@ -6,65 +6,74 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+resources :books, only: [:show]
+books#show
+
+or did you want the curl request code?
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+routes.rb
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+It defines how the database should organize the hash data (model) into the database.
 ```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```md
-// your response here
+I think it's the admin rights, but not positive.
+Or does scaffold create the model for the things and the controller is created automatically while
+the rails g controller things only makes a controller file of things?
```

Comment:
The first command generates routes, a controller and model for foo. The second generates just a foo controller.